plugins/example_plugin/linux/Makefile

```diff
@@ -0,0 +1,139 @@
+# Copyright 2019 Google LLC
```

Comment:
Is the idea that each plugin essentially has its own Makefile? Would this be something generated by the tool or provided by the developer?

Reply:
This is very much not yet what I expect plugin builds to look like, it's essentially just a Make version of the current (GN) approach. E..g, the macOS version of this for now will have an Xcode project just like all the other plugins currently do, but longer term I don't think that's how they will actually work (since I think we'll follow the iOS model).

If we do have to keep something like this for Linux we'd want it to be part of the `create` template, but that's still TBD.

(I'll retool the plugin README at some point to be clearer about the extent to which example_plugin's build is preliminary, rather than a long-term solution.)